examples/cluster/aws/main.tf

```
@@ -27,11 +27,11 @@ resource "redpanda_cluster" "test" {
   tags = {
     "key" = "value"
   }
-  aws_private_link = {
-    enabled         = true
-    connect_console = true
-    allowed_principals = ["arn:aws:iam::123456789024:root"]
-  }
+  # aws_private_link = {
+  #   enabled         = true
+  #   connect_console = true
+  #   allowed_principals = ["arn:aws:iam::123456789024:root"]
+  # }
```

should these be uncommented or just removed?

---

They're there as documentation of how to set them but can't be uncommented because they're using fake values that will fail validation in the API which breaks the example.